xunit.runner.visualstudio.testadapter/VsTestRunner.cs

```diff
@@ -69,6 +73,11 @@ void ITestDiscoverer.DiscoverTests(IEnumerable<string> sources, IDiscoveryContex
             var stopwatch = Stopwatch.StartNew();
             var loggerHelper = new LoggerHelper(logger, stopwatch);
 
+#if !PLATFORM_DOTNET
+            AppDomain = SetAppDomainSupport(discoveryContext.RunSettings.SettingsXml);
```

Comment:
You cannot disable app domain support during discovery, because RockSteady leaves the DLLs loaded into memory, and that breaks rebuilding the code (unless the user shuts down the discovery process by hand from Task Manager).